packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -90,21 +90,21 @@ function definitionNodeHasName(x: DefinitionNode): x is DefinitionNode & { name:
     return "name" in x;
 }
 
-function makeAugmentedSchema(
-    document: DocumentNode,
-    {
-        features,
-        generateSubscriptions,
-        userCustomResolvers,
-        subgraph,
-    }: {
-        features?: Neo4jFeaturesSettings;
-        generateSubscriptions?: boolean;
-        userCustomResolvers?: IResolvers | Array<IResolvers>;
-        subgraph?: Subgraph;
-    } = {},
-    schemaModel: Neo4jGraphQLSchemaModel
-): {
+function makeAugmentedSchema({
+    document,
+    features,
+    userCustomResolvers,
+    subgraph,
+    schemaModel,
+    _experimental,
+}: {
+    document: DocumentNode;
+    features?: Neo4jFeaturesSettings;
+    userCustomResolvers?: IResolvers | Array<IResolvers>;
+    subgraph?: Subgraph;
+    schemaModel: Neo4jGraphQLSchemaModel;
+    _experimental: boolean;
```

Comment:
I'm assuming this is because it's currently unused - this underscore will be removed once it is used?

Reply:
yes, that is indeed the plan :D